src/python/bot/tasks/fuzz_task.py

```diff
@@ -63,14 +63,21 @@
 from system import process_handler
 from system import shell
 
+SelectionMethod = namedtuple('SelectionMethod', 'method_name probability')
+
 FUZZ_TARGET_UPDATE_FAIL_RETRIES = 5
 FUZZ_TARGET_UPDATE_FAIL_DELAY = 2
 DEFAULT_CHOOSE_PROBABILITY = 9  # 10%
 FUZZER_METADATA_REGEX = re.compile(r'metadata::(\w+):\s*(.*)')
 FUZZER_FAILURE_THRESHOLD = 0.33
 MAX_GESTURES = 30
 MAX_NEW_CORPUS_FILES = 500
-MULTI_ARMED_BANDIT_TRAFFIC_SPLIT = .2
+SELECTION_METHOD_DISTRIBUTION = [
+    SelectionMethod('default', .55),
```

Comment:
nit: maybe 20-20-20-40? That way you'd get a decent amount of data faster, plus 2x different versus the default selection might be helpful if we need to somehow compare any absolute numbers.

Reply:
will talk to marty about this